### PR TITLE
[zh-cn]: update the translation of throw statement

### PR DESCRIPTION
**Pull request by T34-active**
### Related issues and pull requests
https://github.com/mdn/translated-content/issues/18266

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/statements/throw/index.md [hunk 1]
```diff
@@ -1,162 +1,122 @@
...
 title: throw
 slug: Web/JavaScript/Reference/Statements/throw
+l10n:
+  sourceCommit: 4c26e8a3fb50d06963b06017f51ce19364350564
 ---
 
 {{jsSidebar("Statements")}}
 
-**`throw`** **语句**用来抛出一个用户自定义的异常。当前函数的执行将被停止（`throw` 之后的语句将不会执行），并且控制将被传递到调用堆栈中的第一个 [`catch`](/zh-CN/docs/Web/JavaScript/Reference/Statements/try...catch) 块。如果调用者函数中没有 `catch` 块，程序将会终止。
+**`throw`** 语句用于抛出用户自定义异常。当前函数的执行将停止（`throw` 之后的语句不会被执行），并且控制权将传递给调用堆栈中第一个[`catch`](/zh-CN/docs/Web/JavaScript/Reference/Statements/try...catch)块。如果调用函数中没有 `catch` 块，则程序将终止。
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/statements/throw/index.md:10]:
```suggestion
**`throw`** 语句用于抛出用户自定义异常。当前函数的执行将停止（`throw` 之后的语句不会被执行），并且控制权将传递给调用堆栈中第一个 [`catch`](/zh-CN/docs/Web/JavaScript/Reference/Statements/try...catch) 块。如果调用函数中没有 `catch` 块，则程序将终止。
```

File: files/zh-cn/web/javascript/reference/statements/throw/index.md [hunk 2]
```diff
@@ -1,162 +1,122 @@
...
+console.log(sum(1, 2, 3)); // 6
 try {
-  // statements to try
-  var myMonth = 15; // 15 超出边界并引发异常
-  var monthName = getMonthName(myMonth);
+  sum("1", "2");
 } catch (e) {
-  var monthName = "unknown";
-  console.log(e.message, e.name); // 传递异常对象到错误处理
+  console.error(e); // TypeError: 只能添加数字
```
Comment (skyclouds2001) [inline files/zh-cn/web/javascript/reference/statements/throw/index.md:84]:
```suggestion
  console.error(e); // TypeError：只能添加数字
```

File: files/zh-cn/web/javascript/reference/statements/throw/index.md [hunk 3]
```diff
@@ -1,162 +1,122 @@
...
 title: throw
 slug: Web/JavaScript/Reference/Statements/throw
+l10n:
+  sourceCommit: 4c26e8a3fb50d06963b06017f51ce19364350564
 ---
 
 {{jsSidebar("Statements")}}
 
-**`throw`** **语句**用来抛出一个用户自定义的异常。当前函数的执行将被停止（`throw` 之后的语句将不会执行），并且控制将被传递到调用堆栈中的第一个 [`catch`](/zh-CN/docs/Web/JavaScript/Reference/Statements/try...catch) 块。如果调用者函数中没有 `catch` 块，程序将会终止。
+**`throw`** 语句用于抛出用户自定义异常。当前函数的执行将停止（`throw` 之后的语句不会被执行），并且控制权将传递给调用堆栈中第一个 [`catch`](/zh-CN/docs/Web/JavaScript/Reference/Statements/try...catch) 块。如果调用函数中没有 `catch` 块，则程序将终止。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/throw/index.md:10]:
```suggestion
**`throw`** 语句用于抛出用户自定义的异常。当前函数的执行将停止（`throw` 之后的语句不会被执行），并且控制权将传递给调用堆栈中第一个 [`catch`](/zh-CN/docs/Web/JavaScript/Reference/Statements/try...catch) 块。如果调用函数中没有 `catch` 块，则程序将终止。
```

File: files/zh-cn/web/javascript/reference/statements/throw/index.md [hunk 4]
```diff
@@ -1,162 +1,122 @@
...
 ```
 
 - `expression`
-  - : 要抛出的表达式。
+  - : 抛出表达式。
 
 ## 描述
 
-使用`throw`语句来抛出一个异常。当你抛出异常时，`expression` 指定了异常的内容。下面的每行都抛出了一个异常：
+`throw` 语句用于抛出一个异常，该异常会沿着调用堆栈向上传播。有关错误冒泡和处理的更多信息，请参阅[流程控制与错误处理](/zh-CN/docs/Web/JavaScript/Guide/Control_flow_and_error_handling)。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/throw/index.md:25]:
```suggestion
`throw` 语句在任何可以使用语句的上下文中都有效。它的执行会产生一个在调用堆栈传播的异常。有关错误冒泡和处理的更多信息，请参阅[流程控制与错误处理](/zh-CN/docs/Web/JavaScript/Guide/Control_flow_and_error_handling)。
```

File: files/zh-cn/web/javascript/reference/statements/throw/index.md [hunk 5]
```diff
@@ -1,162 +1,122 @@
...
+throw error; // 抛出之前定义的值（例如，在 catch 块中）
+throw new Error("Required"); // 抛出一个新的错误对象
 ```
 
-注意`throw`语句同样受到[自动分号插入（ASI](/zh-CN/docs/Web/JavaScript/Reference/Lexical_grammar#Automatic_semicolon_insertion)）机制的控制，在`throw`关键字和值之间不允许有行终止符。
+在实践中，你抛出的异常应该*始终*是 {{jsxref("Error")}} 对象或 `Error` 子类的实例，例如 {{jsxref("RangeError")}}。这是因为捕获错误的代码可能期望捕获的值具有一些属性，例如 {{jsxref("Error/message", "错误信息")}}。例如，Web API 通常会抛出 {{domxref("DOMException")}} 实例，这些实例继承自 `Error.prototype`。
+
+### 自动分号补全（ASI）
+
+语法不允许在 `throw` 关键字和要抛出的表达式之间出现行终止符。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/throw/index.md:38]:
```suggestion
语法不允许在 `throw` 关键字和要抛出的表达式之间出现换行。
```

File: files/zh-cn/web/javascript/reference/statements/throw/index.md [hunk 6]
```diff
@@ -1,162 +1,122 @@
...
-throw 42; // 抛出了一个值为整数 42 的异常
-throw true; // 抛出了一个值为 true 的异常
+throw error; // 抛出之前定义的值（例如，在 catch 块中）
+throw new Error("Required"); // 抛出一个新的错误对象
 ```
 
-注意`throw`语句同样受到[自动分号插入（ASI](/zh-CN/docs/Web/JavaScript/Reference/Lexical_grammar#Automatic_semicolon_insertion)）机制的控制，在`throw`关键字和值之间不允许有行终止符。
+在实践中，你抛出的异常应该*始终*是 {{jsxref("Error")}} 对象或 `Error` 子类的实例，例如 {{jsxref("RangeError")}}。这是因为捕获错误的代码可能期望捕获的值具有一些属性，例如 {{jsxref("Error/message", "错误信息")}}。例如，Web API 通常会抛出 {{domxref("DOMException")}} 实例，这些实例继承自 `Error.prototype`。
+
+### 自动分号补全（ASI）
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/throw/index.md:36]:
```suggestion
### 自动分号补全
```

File: files/zh-cn/web/javascript/reference/statements/throw/index.md [hunk 7]
```diff
@@ -1,162 +1,122 @@
...
+```js-nolint
+throw (
+  new Error()
+);
+```
 
 ## 示例
 
-### 抛出一个对象
+### 抛出用户自定义错误
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/throw/index.md:64]:
```suggestion
### 抛出用户自定义的错误
```

**Review by JasonLamv-t — COMMENTED** [review]
Good job. Let's make some improvement.

**Review by JasonLamv-t — APPROVED** [review]
LGTM. Thank you for your work! @T34-active

**Pull request merged**